Makefile

```diff
@@ -129,3 +131,16 @@ local_gems_bundle:
 
 local_gems_run: local_gems_bundle
 	BUNDLE_GEMFILE=Gemfile-dev make run
+
+build_artifact:
+	bundle config set --local cache_all true
+	bundle package
+	tar --exclude './config/agencies.yml' --exclude './config/iaa_gtcs.yml' \
+		--exclude './config/iaa_orders.yml' --exclude './config/iaa_statuses.yml' \
+		--exclude './config/integration_statuses.yml' --exclude './config/integrations.yml' \
+		--exclude './config/partner_account_statuses.yml' --exclude './config/partner_accounts.yml' \
+		--exclude './config/service_providers.yml' --exclude='./certs/sp' \
+		--exclude='./identity-idp-config' --exclude='./tmp' --exclude='./node_modules' \
```

Comment:
personally I'd try to fit all these on one line or break them all onto their own lines? I know it's a ton of lines but we already have a long makefile so 🤷

Reply:
I broke them up into their own lines for readability